credsweeper/credentials/line_data.py

```diff
@@ -32,13 +32,15 @@ def __init__(
             self,  #
             config: Config,  #
             line: str,  #
+            line_pos: int,  #
```

Comment:
Could you please explain more about `line_pos`?
As I read the code it is very similar with `line_num`.

Reply:
line_pos is used in the class for support MultiPattern
old: https://github.com/Samsung/CredSweeper/blob/3989a25b8c40527aece9c63a54008070ef39d7a3/credsweeper/scanner/scan_type/multi_pattern.py#L45
- it may be wrong when e.g. xml was parsed with "sieve" numeration. 
Additionally, diff parsing has to used emply lines to support "line_num" which is an information only. It taked extra memory too.

Reply:
I understood about it and concerns.
Thank you for explanation.